py/src/braintrust/logger.py

```diff
@@ -888,6 +889,11 @@ def decorator(span_args, span_kwargs, f):
 
         f_sig = inspect.signature(f)
 
+        if "span_attributes" not in span_kwargs:
```

Comment:
What if they pass `span_attributes` as a positional argument? If we wanted to be very strict about making it a keyword-only argument, we could add an asterisk to the function signature, so it becomes `def start_span(name=None, *, span_attributes={}, ...)`.